pkg/materialize/format_specs.go

```diff
@@ -89,10 +106,42 @@ func GetSinkFormatSpecStruc(v interface{}) SinkFormatSpecStruct {
 		if csr, ok := avro.([]interface{})[0].(map[string]interface{})["schema_registry_connection"]; ok {
 			key := avro.([]interface{})[0].(map[string]interface{})["avro_key_fullname"].(string)
 			value := avro.([]interface{})[0].(map[string]interface{})["avro_value_fullname"].(string)
+
+			var docType AvroDocType
+			if adt, ok := avro.([]interface{})[0].(map[string]interface{})["avro_doc_type"].([]interface{}); ok {
```

Comment:
I think that the tests are failing due to this line here. As far as I can tell `adt[0]` is accessed without checking if adt (which is expected to be a slice) has any elements. 

```suggestion
			if adt, ok := avro.([]interface{})[0].(map[string]interface{})["avro_doc_type"].([]interface{}); ok && len(adt) > 0 {
```